src/property.h

```diff
@@ -40,6 +40,8 @@
 
 namespace quadiron {
 
+#define OOR_MARK 1
```

Comment:
We could use a `static const int` (or `static constexpr int`)

Reply:
Could even be a constexpr I think. (just replace `const` by `constexpr`).

`constexpr` can be used in more context than `const`, so they should be preferred when possible ;)